crates/ruff_formatter/src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl LineWidth {
 
 impl Default for LineWidth {
     fn default() -> Self {
-        Self(80)
+        Self(88)
```

Comment:
i think it makes sense to use the correct default here already

Reply:
I would prefer keeping 80, although it's hard reasoning for it if it complicates the implementation. 
The reason I prefer 80 is because `ruff_formatter` is language agnostic and 88 is a very python specific line width. 

Most other languages prefer to default to 80.